source/Target/SwiftLanguageRuntime.cpp

```diff
@@ -1926,14 +1929,17 @@ bool SwiftLanguageRuntime::GetDynamicTypeAndAddress_Protocol(
   lldb::addr_t existential_address;
   bool use_local_buffer = false;
 
-  if (in_value.GetValueType() == eValueTypeConstResult) {
+  if (in_value.GetValueType() == eValueTypeConstResult &&
```

Comment:
This condition alone was just a proxy for what we actually wanted to test, which is in the next line.

Reply:
I thought we were going to replace this with `DataExtractor`? Are you doing it as a follow up?

Reply:
It turned out the the `else` code path is already reading memory from the process.

Reply:
Very good, thanks.